Src/CSharpier/Utilities/StringDiffer.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.IO;
+
+namespace CSharpier.Utilities
+{
+    public static class StringDiffer
+    {
+        public static void PrintDifference(string expected, string actual, IConsole console)
+        {
+            if (expected == actual)
+            {
+                return;
+            }
+
+            using var expectedReader = new StringReader(expected);
+            using var actualReader = new StringReader(actual);
+
+            var expectedLine = expectedReader.ReadLine();
+            var actualLine = actualReader.ReadLine();
+            var line = 1;
+            string? previousExpectedLine = null;
+            string? previousActualLine = null;
+            while (expectedLine != null || actualLine != null)
+            {
+                if (expectedLine == actualLine)
+                {
+                    line++;
+                    previousExpectedLine = expectedLine;
+                    previousActualLine = actualLine;
+                    expectedLine = expectedReader.ReadLine();
+                    actualLine = actualReader.ReadLine();
+                    continue;
+                }
+
+                console.WriteLine(
+                    $"----------------------------- Expected: Around Line {line} -----------------------------"
+                );
+                if (previousExpectedLine != null)
+                {
+                    console.WriteLine(MakeWhiteSpaceVisible(previousExpectedLine));
+                }
+                console.WriteLine(MakeWhiteSpaceVisible(expectedLine));
+                var nextExpectedLine = expectedReader.ReadLine();
```

Comment:
Maybe leave a comment somewhere in/around this function that if there are multiple lines that are different, it only prints around the first divergence. That would have made this code much quicker to parse.